Source/WebCore/Modules/webaudio/AudioWorkletMessagingProxy.cpp

```diff
@@ -97,24 +97,24 @@ RefPtr<CacheStorageConnection> AudioWorkletMessagingProxy::createCacheStorageCon
 RefPtr<RTCDataChannelRemoteHandlerConnection> AudioWorkletMessagingProxy::createRTCDataChannelRemoteHandlerConnection()
 {
     ASSERT(isMainThread());
-    if (!m_document->page())
+    if (!m_document.get().page())
```

Comment:
All these `.get()` changes shouldn't be needed, you can totally use `->` with a WeakRef or a WeakPtr or a CheckedRef or a CheckedPtr. Please revert them.